src/components/activity/ActivityFormPanel.svelte

```diff
@@ -54,7 +54,8 @@
   let formParameters: FormParameter[] = [];
   let isChild: boolean;
   let model: Model;
-  let parameterError: string | null = null;
+  let parametersWithErrorsCount: number = 0;
+  let parameterErrorMap: { [key: string]: string[] } = {};
```

Comment:
Can this type just be `Record<string[]>`?

Reply:
Yup, changing to `Record<string, string[]>`